client/js/index.js

```diff
@@ -0,0 +1,59 @@
+const allPagesButton = document.querySelector('#all-pages-button');
+const randomPageButton = document.querySelector('#random-page-button');
+const textInput = document.querySelector('input');
+
+initListeners();
+
+function getSearchInput () {
+    return textInput.value;
+}
+
+function initListeners() {
+    allPagesButton.addEventListener('click', e => search(e));
+    randomPageButton.addEventListener('click', randomSearch);
+}
+
+
+function search(e){
+    e.preventDefault();
+    const input = getSearchInput();
+    console.log(input)
+
+    // if text exists
+    if (input) {
+        // fetchQuery();
+        localStorage.setItem("query", input);
+        window.location.href = './results.html';
+    } else {
+        // do something
+    }
+}
+
+function randomSearch(){
+    const luckyText = getSearchInput()
+
+    if (luckyText) {
+        if (luckyText.toLowerCase() == "artists") {
+            getRandomArtist()
+        } else {
+            getArtistWiki(luckyText)
+        }
+    } else {
+        getRandomArtist();
+    }
+}
```

Comment:
Function doesn't let me randomly search unless it's just 'artists' or a specific name or nothing in the search bar - doesn't return anything for something like artistsmale